src/base/PolymeshTransactionBase.ts

```diff
@@ -646,6 +653,32 @@ export abstract class PolymeshTransactionBase<
     }
   }
 
+  /**
+   * returns the transaction result - this is the same value as the Promise run returns
+   * @note it is generally preferable to `await` the `Promise` returned by { @link base/PolymeshTransactionBase!PolymeshTransactionBase.run | transaction.run() } over reading this property
+   *
+   * @throws if the is { @link base/PolymeshTransactionBase!PolymeshTransactionBase.isSuccess | transaction.isSuccess } is false — be sure to check before accessing
```

Comment:
Some word is missing here

Reply:
Good catch. I think its actually left over is from "isSuccess"